README.md

```diff
@@ -1,18 +1,41 @@
 # Azure Exporter for Scrapy
+scrapy_azure_exporter is a Python package which can be used in integration with scrapy for exporting feeds to Azure Storage.
+## Description:
+The scrapy_azure_exporter takes inspiration from the built-in feed exporters in scrapy. The built-in exporters doesn't have support for Azure yet.
+So this scrapy_azure_exporter can be used whenever when we need the feeds to be exported to Azure Blob storage.
```

Comment:
```suggestion
[Scrapy feed export storage backend](https://doc.scrapy.org/en/latest/topics/feed-exports.html#storage-backends) for [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/).
```

- No need to have a separate description section, the first paragraph(s) should be the description.
- The shorter, the better.